.github/issue-branch.yml

```diff
@@ -10,4 +10,6 @@ branches:
     prefix: epic/
   - label: type/docs
     prefix: docs/
+  - label: type/automation
+    prefix: automation/
```

Comment:
Pls change to auto/ - less characters to type when I'm switching branches :)

Reply:
which line 13 or 14?

Reply:
14